cdap-client/src/main/java/co/cask/cdap/client/util/VersionMigrationUtils.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.client.util;
+
+import co.cask.cdap.client.config.ClientConfig;
+import co.cask.cdap.common.conf.Constants;
+import co.cask.cdap.proto.ProgramType;
+import com.google.common.base.Preconditions;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+/**
+ * Utility methods used by CDAP Clients for CDAP API version migration
+ */
+public class VersionMigrationUtils {
```

Comment:
Should these methods be in ``RESTMigrationUtils`` in ``cdap-common``. Don't feel too strongly, but would like to keep static helper classes to the minimum.